src/assets/simp/build/simp.spec

```diff
@@ -23,127 +23,128 @@ Obsoletes: pupmod-simp-activemq <= 4.0.1
 Requires: pupmod-herculesteam-augeasproviders_shellvar >= 3.0.0, pupmod-herculesteam-augeasproviders_shellvar < 4.0.0
 Requires: pupmod-herculesteam-augeasproviders_ssh >= 3.0.0, pupmod-herculesteam-augeasproviders_ssh < 4.0.0
-Requires: pupmod-herculesteam-augeasproviders_sysctl >= 2.1.0-2016, pupmod-herculesteam-augeasproviders_sysctl < 3.0.0
+Requires: pupmod-herculesteam-augeasproviders_sysctl >= 2.2.0, pupmod-herculesteam-augeasproviders_sysctl < 3.0.0
 Requires: pupmod-onyxpoint-gpasswd >= 1.0.5, pupmod-onyxpoint-gpasswd < 2.0.0
 Requires: pupmod-puppet-yum >= 2.2.1, pupmod-puppet-yum < 3.0.0
-Requires: pupmod-puppetlabs-apache >= 1.10.0-2016, pupmod-puppetlabs-apache < 4.0.0
-Requires: pupmod-puppetlabs-concat >= 2.2.0-2016, pupmod-puppetlabs-concat < 5.0.0
+Requires: pupmod-puppetlabs-apache >= 3.0.0, pupmod-puppetlabs-apache < 4.0.0
+Requires: pupmod-puppetlabs-concat >= 4.1.1, pupmod-puppetlabs-concat < 5.0.0
 Requires: pupmod-puppetlabs-hocon >= 1.0.0, pupmod-puppetlabs-hocon < 2.0.0
-Requires: pupmod-puppetlabs-inifile >= 1.6.0-2016, pupmod-puppetlabs-inifile < 3.0.0
-Requires: pupmod-puppetlabs-java >= 1.2.0-2016, pupmod-puppetlabs-java < 3.0.0
+Requires: pupmod-puppetlabs-inifile >= 2.2.0, pupmod-puppetlabs-inifile < 3.0.0
+Requires: pupmod-puppetlabs-java >= 2.4.0, pupmod-puppetlabs-java < 3.0.0
 Requires: pupmod-puppetlabs-motd >= 1.9.0, pupmod-puppetlabs-motd < 2.0.0
 Requires: pupmod-puppetlabs-mount_providers >= 1.0.0, pupmod-puppetlabs-mount_providers < 2.0.0
 Requires: pupmod-puppetlabs-postgresql >= 5.4.0, pupmod-puppetlabs-postgresql < 6.0.0
-Requires: pupmod-puppetlabs-puppetdb >= 5.0.0, pupmod-puppetlabs-puppetdb < 7.0.0
-Requires: pupmod-puppetlabs-puppet_authorization >= 0.2.0, pupmod-puppetlabs-puppet_authorization < 1.0.0
+Requires: pupmod-puppetlabs-puppetdb >= 6.0.2, pupmod-puppetlabs-puppetdb < 7.0.0
+Requires: pupmod-puppetlabs-puppet_authorization >= 0.4.0, pupmod-puppetlabs-puppet_authorization < 1.0.0
 Requires: pupmod-puppetlabs-stdlib >= 4.25.1, pupmod-puppetlabs-stdlib < 5.0.0
 Requires: pupmod-richardc-datacat >= 0.6.2, pupmod-richardc-datacat < 1.0.0
-Requires: pupmod-simp-acpid >= 1.0.1, pupmod-simp-acpid < 2.0.0
-Requires: pupmod-simp-aide >= 6.0.0, pupmod-simp-aide < 7.0.0
-Requires: pupmod-simp-at >= 0.0.1, pupmod-simp-at < 1.0.0
-Requires: pupmod-simp-auditd >= 7.0.0, pupmod-simp-auditd < 9.0.0
-Requires: pupmod-simp-autofs >= 6.0.0, pupmod-simp-autofs < 7.0.0
-Requires: pupmod-simp-chkrootkit >= 0.0.1, pupmod-simp-chkrootkit < 1.0.0
-Requires: pupmod-simp-clamav >= 6.0.0-2016, pupmod-simp-clamav < 7.0.0
-Requires: pupmod-simp-compliance_markup >= 2.0.0, pupmod-simp-compliance_markup < 3.0.0
-Requires: pupmod-simp-cron >= 0.0.1, pupmod-simp-cron < 1.0.0
-Requires: pupmod-simp-dhcp >= 6.0.0-2016, pupmod-simp-dhcp < 7.0.0
-Requires: pupmod-simp-fips >= 0.1.0, pupmod-simp-fips < 1.0.0
-Requires: pupmod-simp-freeradius >= 7.0.0-2016, pupmod-simp-freeradius < 8.0.0
```

Comment:
This migration and that of pupmod-simp-krb5 should be noted in the %changelog below and in the Changelog.rst and/or upgrade guide.

Reply:
Since pupmod-simp-nfs requires upmod-simp-krb5, it doesn't seem like pupmod-simp-krb5 should be moved to extras

Reply:
pupmod-simp-simp requires pupmod-simp-freeradius, so probably don't want to move to simp-extras afterall